mdbook-course/src/course.rs

```diff
@@ -394,28 +382,19 @@ impl Segment {
         self.into_iter().map(|s| s.minutes()).sum()
     }
 
-    pub fn outline(&self, at_source_path: impl AsRef<Path>) -> String {
-        let mut outline = String::from("In this segment:\n");
+    pub fn outline(&self) -> String {
+        let mut slides = Table::new(["Slide".into(), "Time".into()]);
```

Comment:
Would it be slightly more accurate to call this a "duration"?

```suggestion
        let mut slides = Table::new(["Slide".into(), "Duration".into()]);
```